src/components/SearchBar/SearchBar.jsx

```diff
@@ -0,0 +1,361 @@
+import React, { Component } from "react";
```

Comment:
this search bar doesn't take in consideration the pending bills  yet right

Reply:
It does not.